### ISSUE TITLE
Polly Issues

### ISSUE THREAD

**Issue by leitesilvavitor** [karashiiro/TextToTalk#145]
Hello, I'm currently having trouble when using Polly as a backend. It was working until last week, but since then there are no voices to select from. I thought to wait for a bit and see if it would be fixed with updates, but I continue to have the same issue even after v1.21.7.0, which refers to the issues I'm having.

Since this is the first time I'm opening an issue, please let me know if there is missing information I should provide.

<img width="188" alt="image" src="https://user-images.githubusercontent.com/98501774/218766264-07e78691-10d2-4e96-ba58-323852b7b990.png">
<img width="252" alt="image" src="https://user-images.githubusercontent.com/98501774/218766341-f6a5fdb9-a6a7-4c34-9b87-6e3601b0329f.png">

**Comment by karashiiro** [comment]
It looks like you don't have access keys set; those might have been cleared out at some point. Try restoring those (or [generating new ones](https://github.com/karashiiro/TextToTalk/wiki/Creating-access-keys-for-Amazon-Polly)) and then see if it works.

If it doesn't, send your `dalamud.log` file (at `%AppData%\XIVLauncher\dalamud.log`) and I can see what the specific issue is.

**Comment by leitesilvavitor** [comment]
Thank you for the help, I made new credentials and now I can select the voices for the standard engine, but still no voices are available for the Neural engine, which was what I was using before.

Since in the logs the error appears to be related to the region selected, I tried changing the regions (sa-east-1, us-east-2, us-west-2) and repeating the process, but the same error relating to the region is repeated on the logs.
I'll attach the log below for more detailed information.

[dalamud.log](https://github.com/karashiiro/TextToTalk/files/10733568/dalamud.log)

Thanks again!
